draft-ietf-rats-architecture.md

```diff
@@ -376,6 +376,76 @@ create Evidence. Examples of environments that can be used as
 Attesting Environments include Trusted Execution Environments (TEE),
 embedded Secure Elements (eSE), or Hardware Security Modules (HSM).
 
+## Layered Attestation Procedures
+
+By definition, the Attester role takes on the duty to create Attestation Evidence.
+The fact that an Attester role is composed of several types of environments that
+can be nested or staged adds complexity to the architectural layout of how an
```

Comment:
not sure what "staged" means

Reply:
Introduced the term Staging at the end of the 3rd paragraph. I think this is better than not using "staging", it is an intuitive term that does not has to be introduces, necessarily. Any other thoughts?